src/pages/homeCollection/printLabels.js

```diff
@@ -0,0 +1,138 @@
+                    <div class="card">
+                      <div class="card-body">
+                      <span> <h3 style="text-align: center; margin: 0 0 1rem;">How many labels to print?</h3> </span>
+                        <div style="text-align: center;  padding-bottom: 25px; "> 
+                          <input required type="text" name="numberToPrint" id="numberToPrint"  /> 
+                        </div>
+                        <span> Labels to print: ${ appState.getState().length }  </span>
+                        <br />
+                        <div class="mt-4 mb-4" style="display:inline-block;">
+                          <button type="button" class="btn btn-primary" id="clearForm" disabled>View All Printed Labels</button>
+                          <button type="submit" class="btn btn-primary" id="generateCsv">Download CSV File</button>
+                        </div>
+                        </div>
+                      </div>
+                  </div>
+                    
+                  </div>
+                </div>`;
+
+  template += `
+        <div style="overflow:auto; height:45vh">
+        </div>`;
+
+  document.getElementById("navbarNavAltMarkup").innerHTML = nonUserNavBar(name);
+  contentBody.innerHTML = template;
+  activeHomeCollectionNavbar();
+
+  generateParticipantCsvGetter(name);
+};
+
+const initializeTotalAddressesToPrint = async () => {
+  showAnimation();
+  const totalAddresses = await getTotalAddressesToPrint();
+  appState.setState({'totalAddresses': totalAddresses.data})
+  appState.setState({'length': totalAddresses.data.length })
+  hideAnimation();
+  return totalAddresses.data.length
+}
+
+const getTotalAddressesToPrint = async () => {
+  const idToken = await getIdToken();
+  const response = await fetch(`${baseAPI}api=totalAddressesToPrint`, {
+      method: "GET",
+      headers: {
+          Authorization:"Bearer "+idToken
+      }
+  });
+  return await response.json();
+}
+
+const generateParticipantCsvGetter = (name) => {
+  const a = document.getElementById("generateCsv");
+  if (a) {
+      a.addEventListener("click", () => {
+        const numberToPrint = document.getElementById("numberToPrint").value;
+          if (numberToPrint) {
+            const arrayLengthToProcess = appState.getState().length
+            if (arrayLengthToProcess >= numberToPrint) {
+              const arrayToProcess = appState.getState().totalAddresses
+              const remainingArrayToProcess = arrayToProcess.slice(numberToPrint, appState.getState().length)
+              appState.setState({totalAddresses: remainingArrayToProcess })
+              appState.setState({'length': remainingArrayToProcess.length })
+              generateParticipantCsv(arrayToProcess.slice(0, numberToPrint));
+              printLabelsTemplate(name);
+                // Display success message
+              const alertList = document.getElementById('alert_placeholder');
+              const template = `
```

Comment:
NIT: Should we assign the string value for alertList.innerHTML without declaring `template`